internal/shared/api/handlers/handler/supply.go

```diff
@@ -34,7 +34,7 @@ func (h *Handler) babyTotalSupply(req *http.Request) (any, error) {
 	}
 	ctx := req.Context()
 
-	coin, err := h.bbnClient.GetTotalSupply(ctx, "ubbn")
```

Comment:
Always wanted to start renaming `Get*` methods (according to official go style getters mustn't start with "Get" - https://go.dev/doc/effective_go#Getters)